pact_broker/docker_configuration.rb

```diff
@@ -128,6 +132,35 @@ def to_s
       end
     end
 
+    def space_delimited_integer_list_or_default property_name
+      if env_populated?(property_name)
+        SpaceDelimitedIntegerList.parse(env(property_name))
+      else
+        default(property_name)
+      end
+    end
+
+    class SpaceDelimitedIntegerList < Array
+      def initialize list
+        super(list)
+      end
+
+      def self.parse(string)
+        array = (string || '').split(' ').filter { |word|
+          (Integer(word) rescue nil) != nil
+        }.collect { | word |
+          word.to_i
+        }
+        SpaceDelimitedIntegerList.new(array)
+      end
+
+      def to_s
+        collect do | integer |
+          integer.to_s
+        end.join(' ')
```

Comment:
```suggestion
        collect(&:to_s).join(' ')
```

Reply:
good advice. I was not aware of this method reference. Thanks.